client/src/app/pages/main/nav/nav.component.ts

```diff
@@ -55,4 +55,17 @@ export class NavComponent implements OnInit {
       }
     });
   }
+
+  scrollTo(elementID:string): boolean {
+    switch(elementID){
+      case 'about':
+      case 'gamemodes':
+      case 'footer':
+        document.getElementById(`${elementID}`).scrollIntoView();
```

Comment:
Let's pass in
```suggestion
        document.getElementById(`${elementID}`).scrollIntoView({ behavior: 'smooth' });
```
It defaults to auto, but I'm not sure auto would be smooth here.